From ab1179d0d4e80520412f7045433551c846465762 Mon Sep 17 00:00:00 2001
From: mydongistiny <jaysonedson@gmail.com>
Date: Sat, 1 Sep 2018 02:33:12 -0700
Subject: [PATCH 13/15] SystemUI: Update string for add tiles with one click

In 9.0 it shows a string in the qs panel telling you how to add/remove tiles and after 'SystemUI: Allow user to add/remove QS with one click' it still says "Hold and drag to add tiles". Update string to the correct way.

Change-Id: I2912e9254f9ed14436119addc913fe9907dc7f4e
---
 packages/SystemUI/res/values-en-rAU/strings.xml | 2 +-
 packages/SystemUI/res/values-en-rCA/strings.xml | 2 +-
 packages/SystemUI/res/values-en-rGB/strings.xml | 2 +-
 packages/SystemUI/res/values-en-rIN/strings.xml | 2 +-
 packages/SystemUI/res/values-en-rXC/strings.xml | 2 +-
 packages/SystemUI/res/values/strings.xml        | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/packages/SystemUI/res/values-en-rAU/strings.xml b/packages/SystemUI/res/values-en-rAU/strings.xml
index 2c9d49510f46..528ce57cab41 100644
--- a/packages/SystemUI/res/values-en-rAU/strings.xml
+++ b/packages/SystemUI/res/values-en-rAU/strings.xml
@@ -629,7 +629,7 @@
     <string name="right_keycode" msgid="2480715509844798438">"Right keycode"</string>
     <string name="left_icon" msgid="5036278531966897006">"Left icon"</string>
     <string name="right_icon" msgid="1103955040645237425">"Right icon"</string>
-    <string name="drag_to_add_tiles" msgid="8933270127508303672">"Hold and drag to add tiles"</string>
+    <string name="drag_to_add_tiles" msgid="230586591689084925">"Press on tiles to add or remove"</string>
     <string name="drag_to_rearrange_tiles" msgid="2143204300089638620">"Hold and drag to rearrange tiles"</string>
     <string name="drag_to_remove_tiles" msgid="4682194717573850385">"Drag here to remove"</string>
     <string name="drag_to_remove_disabled" msgid="933046987838658850">"You need at least <xliff:g id="MIN_NUM_TILES">%1$d</xliff:g> tiles"</string>
diff --git a/packages/SystemUI/res/values-en-rCA/strings.xml b/packages/SystemUI/res/values-en-rCA/strings.xml
index 5abb16cd6443..1f3fde4248a5 100644
--- a/packages/SystemUI/res/values-en-rCA/strings.xml
+++ b/packages/SystemUI/res/values-en-rCA/strings.xml
@@ -629,7 +629,7 @@
     <string name="right_keycode" msgid="2480715509844798438">"Right keycode"</string>
     <string name="left_icon" msgid="5036278531966897006">"Left icon"</string>
     <string name="right_icon" msgid="1103955040645237425">"Right icon"</string>
-    <string name="drag_to_add_tiles" msgid="8933270127508303672">"Hold and drag to add tiles"</string>
+    <string name="drag_to_add_tiles" msgid="230586591689084925">"Press on tiles to add or remove"</string>
     <string name="drag_to_rearrange_tiles" msgid="2143204300089638620">"Hold and drag to rearrange tiles"</string>
     <string name="drag_to_remove_tiles" msgid="4682194717573850385">"Drag here to remove"</string>
     <string name="drag_to_remove_disabled" msgid="933046987838658850">"You need at least <xliff:g id="MIN_NUM_TILES">%1$d</xliff:g> tiles"</string>
diff --git a/packages/SystemUI/res/values-en-rGB/strings.xml b/packages/SystemUI/res/values-en-rGB/strings.xml
index 2c9d49510f46..528ce57cab41 100644
--- a/packages/SystemUI/res/values-en-rGB/strings.xml
+++ b/packages/SystemUI/res/values-en-rGB/strings.xml
@@ -629,7 +629,7 @@
     <string name="right_keycode" msgid="2480715509844798438">"Right keycode"</string>
     <string name="left_icon" msgid="5036278531966897006">"Left icon"</string>
     <string name="right_icon" msgid="1103955040645237425">"Right icon"</string>
-    <string name="drag_to_add_tiles" msgid="8933270127508303672">"Hold and drag to add tiles"</string>
+    <string name="drag_to_add_tiles" msgid="230586591689084925">"Press on tiles to add or remove"</string>
     <string name="drag_to_rearrange_tiles" msgid="2143204300089638620">"Hold and drag to rearrange tiles"</string>
     <string name="drag_to_remove_tiles" msgid="4682194717573850385">"Drag here to remove"</string>
     <string name="drag_to_remove_disabled" msgid="933046987838658850">"You need at least <xliff:g id="MIN_NUM_TILES">%1$d</xliff:g> tiles"</string>
diff --git a/packages/SystemUI/res/values-en-rIN/strings.xml b/packages/SystemUI/res/values-en-rIN/strings.xml
index 2c9d49510f46..528ce57cab41 100644
--- a/packages/SystemUI/res/values-en-rIN/strings.xml
+++ b/packages/SystemUI/res/values-en-rIN/strings.xml
@@ -629,7 +629,7 @@
     <string name="right_keycode" msgid="2480715509844798438">"Right keycode"</string>
     <string name="left_icon" msgid="5036278531966897006">"Left icon"</string>
     <string name="right_icon" msgid="1103955040645237425">"Right icon"</string>
-    <string name="drag_to_add_tiles" msgid="8933270127508303672">"Hold and drag to add tiles"</string>
+    <string name="drag_to_add_tiles" msgid="230586591689084925">"Press on tiles to add or remove"</string>
     <string name="drag_to_rearrange_tiles" msgid="2143204300089638620">"Hold and drag to rearrange tiles"</string>
     <string name="drag_to_remove_tiles" msgid="4682194717573850385">"Drag here to remove"</string>
     <string name="drag_to_remove_disabled" msgid="933046987838658850">"You need at least <xliff:g id="MIN_NUM_TILES">%1$d</xliff:g> tiles"</string>
diff --git a/packages/SystemUI/res/values-en-rXC/strings.xml b/packages/SystemUI/res/values-en-rXC/strings.xml
index 7fde88cb6462..7fe7cbbebc5a 100644
--- a/packages/SystemUI/res/values-en-rXC/strings.xml
+++ b/packages/SystemUI/res/values-en-rXC/strings.xml
@@ -629,7 +629,7 @@
     <string name="right_keycode" msgid="2480715509844798438">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‏‎‏‎‎‎‏‎‎‏‏‎‏‏‎‏‎‏‎‎‎‏‎‏‏‎‎‏‏‏‎‎‏‏‏‎‎‎‎‏‎‎‎‏‏‎‎‏‏‏‏‎‏‎‏‏‏‏‏‎‎‏‏‎‎Right keycode‎‏‎‎‏‎"</string>
     <string name="left_icon" msgid="5036278531966897006">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‏‎‏‏‏‏‎‎‏‎‎‎‏‏‏‎‏‎‎‏‎‏‎‎‏‎‎‎‎‏‏‎‏‏‏‏‏‎‏‎‎‎‎‎‎‎‎‏‎‏‏‎‏‏‎‏‏‏‎‎Left icon‎‏‎‎‏‎"</string>
     <string name="right_icon" msgid="1103955040645237425">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‎‎‏‏‏‏‎‏‎‏‎‎‏‎‎‎‎‎‏‎‎‏‎‏‎‎‎‎‏‎‏‎‏‎‏‏‎‏‏‎‏‎‏‎‎‏‎‏‎‎‏‎‏‎‏‎‏‏‎‎‎‏‎Right icon‎‏‎‎‏‎"</string>
-    <string name="drag_to_add_tiles" msgid="8933270127508303672">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‎‎‏‎‏‎‏‏‎‎‏‏‏‎‏‏‎‎‏‏‎‏‏‏‎‎‎‏‎‏‏‏‏‎‏‏‎‏‎‏‏‏‏‎‎‏‏‏‎‎‎‎Hold and drag to add tiles‎‏‎‎‏‎"</string>
+    <string name="drag_to_add_tiles" msgid="230586591689084925">"Press on tiles to add or remove"</string>
     <string name="drag_to_rearrange_tiles" msgid="2143204300089638620">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‎‏‏‏‏‎‏‏‎‏‏‏‏‏‎‎‎‏‏‎‎‎‎‏‏‏‏‎‏‎‎‏‎‏‎‏‎‏‏‏‏‏‎‏‏‏‎‏‎‏‏‏‏‏‎‏‏‎‏‏‏‎‎‎Hold and drag to rearrange tiles‎‏‎‎‏‎"</string>
     <string name="drag_to_remove_tiles" msgid="4682194717573850385">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‏‏‏‎‎‎‎‎‎‏‏‏‏‏‎‏‎‎‏‏‏‏‏‏‏‎‏‎‎‏‎‎‏‏‏‎‎‏‏‏‎‎‎‏‏‎‎‏‏‎‎‏‎‎‏‎‏‎‎‎‏‎‎‎‏‎Drag here to remove‎‏‎‎‏‎"</string>
     <string name="drag_to_remove_disabled" msgid="933046987838658850">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‎‎‏‏‎‎‏‏‏‏‎‎‏‎‏‏‎‏‏‎‎‏‎‏‎‎‏‎‏‎‏‎‏‏‏‏‎‎‏‏‎‏‏‏‏‏‎‎‎‏‏‎‎‏‎‎‏‎‎‎‏‎‎You need at least ‎‏‎‎‏‏‎<xliff:g id="MIN_NUM_TILES">%1$d</xliff:g>‎‏‎‎‏‏‏‎ tiles‎‏‎‎‏‎"</string>
diff --git a/packages/SystemUI/res/values/strings.xml b/packages/SystemUI/res/values/strings.xml
index e144b43294c6..1a98f58a3a6d 100644
--- a/packages/SystemUI/res/values/strings.xml
+++ b/packages/SystemUI/res/values/strings.xml
@@ -1693,7 +1693,7 @@
     <string name="right_icon">Right icon</string>
 
     <!-- Label for area where tiles can be dragged out of [CHAR LIMIT=60] -->
-    <string name="drag_to_add_tiles">Hold and drag to add tiles</string>
+    <string name="drag_to_add_tiles">Press on tiles to add or remove</string>
 
     <!-- Label for header of customize QS [CHAR LIMIT=60] -->
     <string name="drag_to_rearrange_tiles">Hold and drag to rearrange tiles</string>
-- 
2.34.1

